### ISSUE TITLE
Custom Randomized Function Support

### ISSUE THREAD

**Issue by allenkim** [tf-encrypted/tf-encrypted#818]
Hello,

I had some experiments in mind for some custom activation functions that made use of random noise, but this seems to not be part of standard Keras layers. Thus, I was looking into the more basic conversion from TensorFlow graphs, but wanted to understand if there were ways to do this at a higher level.

If I want to add noise to a tensor during training/inference, can this be done through the Keras layers or would I have to go to a lower level as described [here](https://github.com/tf-encrypted/tf-encrypted/tree/master/tf_encrypted/convert)? At a lower level, is noise supported in this library - I would imagine so if things like dropout are to be supported? Is there a reference I can refer to?

Any help would be appreciated. Thank you!
